components/test_raftstore/src/lib.rs

```diff
@@ -6,6 +6,7 @@ extern crate slog;
 extern crate slog_global;
 #[macro_use]
 extern crate tikv_util;
+extern crate pd as tikv_pd;
```

Comment:
Why not just `pd`?

Reply:
+1

Reply:
because the directory contains a pd.rs file.